src/it/java/edu/iu/uits/mail/SignedEmailIT.java

```diff
@@ -0,0 +1,160 @@
+package edu.iu.uits.mail;
+
+import com.icegreen.greenmail.util.GreenMail;
+import com.icegreen.greenmail.util.GreenMailUtil;
+import com.icegreen.greenmail.util.ServerSetup;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
+
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Enumeration;
+import java.util.Properties;
+
+import static junit.framework.Assert.assertTrue;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertFalse;
+
+public class SignedEmailIT {
+
+    private Properties properties = new Properties();
+    private GreenMail greenMail;
+    private Session session;
+
+
+    private static final int SMTP_PORT = 4025;
+    private static final String SMTP_PROTOCOL = "smtp+smime";
+    private static final String EMAIL_ADDRESS_WITH_CERT = "workflow.noreply@iu.edu";
+    private static final String EMAIL_ADDRESS_WITHOUT_CERT = "workflow.nocert.iu.edu";
+
+    @Before
+    public void setup() throws IOException {
+        final InputStream input = new FileInputStream("/opt/j2ee/security/kr/rice-keystore.properties");
```

Comment:
Since this project is publicly available, do we want to make this location (and the address below) configurable from, say, the environment so outside parties can run these tests? If so, we should probably rename `riceProperties` as well.